201710/isgroba/Caldeiro.java

```diff
@@ -0,0 +1,84 @@
+
+public class Caldeiro {
+
+	private int capacidade;
+	private int contido;
+
+	public Caldeiro() {

```

Comment:
Este constructor hace 3 cosas que el propio compilador ya te hace por defecto y podrías ahorrar las líneas:
- If a constructor does not explicitly invoke a superclass constructor, the Java compiler automatically inserts a call to the no-argument constructor of the superclass. If the super class does not have a no-argument constructor, you will get a compile-time error. Object does have such a constructor, so if Object is the only superclass, there is no problem. (https://docs.oracle.com/javase/tutorial/java/IandI/super.html)

- Los atributos en java se inicializan siempre con 0 (para numeros primitivos), false (para boolean primitivo) y null para objetos. En tu caso, capacidade y contido son enteros primitivos y por lo tanto, son automaticamente inicializados con cero. https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html

Reply:
Grazas polos consellos!! Tereinos presentes!!